app-drawer/app-drawer.html

```diff
@@ -303,22 +319,24 @@
         this._updateDocScroll();
       },
 
-      _positionChanged: function() {
-        this.fire('app-drawer-position-changed');
-      },
-
       _translateDrawer: function(x) {
         this.transform('translate3d(' + x + 'px, 0, 0)', this.$.contentContainer);
       },
 
       _updateDocScroll: function() {
-        this.shouldEnableDocScroll(this.persistent || !this.opened);
```

Comment:
I thought about this, and I don't think calling this `resetLayout` is correct because this method does not trigger any typical layout changes (e.g. changing `position`, `margin`, etc.). It's goal is to disable scrolling, and although we do hacks that do trigger "layout", it doesn't really adjust the position of elements on screen (as opposed to say `<app-drawer-layout>.resetLayout`, which adjusts the margin of the main content).